pool-payout-zkapp/src/PoolPayout.ts

```diff
@@ -36,15 +36,26 @@ export class Reward extends Struct({
   publicKey: PublicKey,
   rewards: UInt64
 }) { // Have the data concatenated here?
+  static blank(): Reward {
+    return new Reward({
+      index: Field(0),
+      publicKey: PublicKey.empty(),
+      rewards: UInt64.from(0)
+    });
+  }
+  isBlank(): Bool {
+    return this.publicKey.equals(PublicKey.empty());
+  }
 }
 
 export class FeePayout extends Struct({
   numDelegates: Field,
   payout: UInt64
-}) { };
+}) { }
 
-export class Rewards2 extends Struct(
-  [Reward, Reward, Reward, Reward, Reward, Reward, Reward, Reward]) { }
+export class Rewards2 extends Struct({
```

Comment:
Made this a much smaller array to limit errors of a size nature.  No reason it could not be further expanded